samples/bluetooth/mesh/light/CMakeLists.txt

```diff
@@ -18,6 +18,12 @@ target_sources(app PRIVATE
 	src/model_handler.c)
 target_include_directories(app PRIVATE include)
 
+# Preinitialization related to DFU over SMP for nRF52 series
+if(CONFIG_SOC_SERIES_NRF52X)
```

Comment:
It also requires additional configurtaion as far as I have seen

Reply:
If we want support for 53DK it requires a separate configuration for the network core that allows using the extended advertiser. Since legacy adv at the moment is the default, such a network core configuration should be conditionally added when using the overlay-dfu.conf. I talked to @m-alperen-sener about it, and while it is possible im concerned that it will be a bit convoluted if we want to add support for 53DK now. When we start using extended advertiser by default I think that this won't be a problem anymore.

Reply:
Ok, lets leave it as is now.